src/jobs/deploy.yml

```diff
@@ -74,4 +78,5 @@ steps:
       bundle-bucket: << parameters.bundle-bucket >>
       bundle-key: << parameters.bundle-key >>
       bundle-type: << parameters.bundle-type >>
-      arguments: << parameters.arguments >>
```

Comment:
this param is still defined on L41

Reply:
Double-checking from the original PR, I think its actually meant to be and my description was wrong. Checking.

Reply:
Yep, its still used, just not here. We may be able to downgrade this to a minor change in fact.

Reply:
Yeah, `arguments` is still used in other steps.

Reply:
Keeping major change as removal of param could affect the command's usage outside of a job